build.mill

```diff
@@ -116,9 +116,9 @@ object Deps {
   val asmTree = ivy"org.ow2.asm:asm-tree:9.7.1"
   val bloopConfig = ivy"ch.epfl.scala::bloop-config:1.5.5"
 
-  val coursierVersion = "2.1.16"
+  val coursierVersion = "2.1.18"
   val coursier = ivy"io.get-coursier::coursier:$coursierVersion"
-  val coursierInterface = ivy"io.get-coursier:interface:1.0.22"
+  val coursierInterface = ivy"io.get-coursier:interface:1.0.24"
```

Comment:
Is there a reason, why you have a lower version in coursier-interface than the version it's wrapping? Do you consider to align these in the future?

Reply:
Understood.